ckan/model/activity.py

```diff
@@ -517,6 +545,24 @@ def _filter_activitites_from_users(q: QActivity) -> QActivity:
 
     return q
 
+
+def _filter_activitites_from_timestamp(
+        q: QActivity,
+        timestamp: datetime.datetime,
+        after: bool
+    ) -> QActivity:
+    '''Adds a filter to an existing query object to include or exclude
+    (include=False) activities based on a list of types.
+
+    '''
+    import ckan.model as model
+    if after:
+        q = q.filter(model.Activity.timestamp > timestamp)
+    else:
+        q = q.filter(model.Activity.timestamp < timestamp)
+    return q
```

Comment:
This kind of 1-line function shouldn't exist because it makes calling code more difficult to understand. Instead please use the line of code itself.